### PR TITLE
add grave

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Cycle for CI

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes ready for review/merge.

File: recipes/grave/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+  home: http://github.com/networkx/grave
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'dead simple graph visualization'
+
+  description: |
+    Grave is a graph visualization package combining ideas from Matplotlib, NetworkX, and seaborn. Its goal is to provide a network drawing API that covers the most use cases with sensible defaults and simple style configuration. Currently, it supports drawing graphs from NetworkX.
```
Comment (ocefpaf) [inline recipes/grave/meta.yaml]: Do you want to break that into multiple lines?

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks @CJ-Wright ! :)

**Pull request merged**